fjord/feedback/models.py

```diff
@@ -617,24 +617,11 @@ def __unicode__(self):
         return unicode(self.id)
 
 
-class NoNullsCharField(serializers.CharField):
-    """Further restricts CharField so it doesn't accept nulls
-
-    DRF lets CharFields take nulls which is not what I want. This
-    raises a ValidationError if the value is a null.
-
-    """
-    def from_native(self, value):
-        if value is None:
-            raise ValidationError('Value cannot be null')
-        return super(NoNullsCharField, self).from_native(value)
-
-
```

Comment:
DRF3 distinguishes between the various possibilities of required, no-nulls, and no-empty-string so we can use that and don't need this `NoNullsCharField` anymore.